<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="pbConDiscon">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbRescan">
          <property name="text">
           <string>Rescan</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbHelp">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbAbout">
          <property name="text">
           <string>About</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbComPort">
        <property name="title">
         <string>COM Port</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbComs">
           <property name="text">
            <string>COMs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbBaudRate">
        <property name="title">
         <string>Baudrate</string>
        </property>
        <layout class="QGridLayout" name="Baudrate">
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButton_5">
           <property name="text">
            <string>9600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="radioButton_7">
           <property name="text">
            <string>19200</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QRadioButton" name="radioButton_8">
           <property name="text">
            <string>28800</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QRadioButton" name="radioButton_6">
           <property name="text">
            <string>14400</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>1200</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QRadioButton" name="radioButton_9">
           <property name="text">
            <string>38400</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="radioButton_10">
           <property name="text">
            <string>56000</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>600</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>2400</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>4800</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="radioButton_11">
           <property name="text">
            <string>57600</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="radioButton_12">
           <property name="text">
            <string>115200</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="radioButton_13">
           <property name="text">
            <string>128000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QRadioButton" name="radioButton_14">
           <property name="text">
            <string>256000</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QRadioButton" name="radioButton_15">
           <property name="text">
            <string>custom</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbDataBits">
        <property name="title">
         <string>Data bits</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QRadioButton" name="radioButton_19">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_18">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_17">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_16">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbParity">
        <property name="title">
         <string>Parity</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QRadioButton" name="radioButton_24">
           <property name="text">
            <string>none</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_23">
           <property name="text">
            <string>odd</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_22">
           <property name="text">
            <string>even</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_21">
           <property name="text">
            <string>mark</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_20">
           <property name="text">
            <string>space</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbStopBits">
        <property name="title">
         <string>Stop bits</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QRadioButton" name="radioButton_27">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_26">
           <property name="text">
            <string>1.5</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_25">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbHandshaking">
        <property name="title">
         <string>Handshaking</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QRadioButton" name="radioButton_29">
           <property name="text">
            <string>none</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_32">
           <property name="text">
            <string>RTS/CTS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_31">
           <property name="text">
            <string>XON/XOFF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_30">
           <property name="text">
            <string>RTS/CTS+XON/XOFF</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_28">
           <property name="text">
            <string>RTS on TX</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gbSettings">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="6">
        <widget class="QPushButton" name="pbASCIITable">
         <property name="text">
          <string>ASCII table</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="cbStreamLog">
         <property name="text">
          <string>Stream log</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="cbAutoStart">
         <property name="text">
          <string>AutoStart Script</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="cbTime">
         <property name="text">
          <string>Time</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="pbGraph">
         <property name="text">
          <string>Graph</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="cbAutoDisConn">
         <property name="text">
          <string>Auto Dis/Connect</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="cbStayOnTop">
         <property name="text">
          <string>Stay on top</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="cbCRLF">
         <property name="text">
          <string>CR=LF</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="pbScripting">
         <property name="text">
          <string>Scripting</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QPushButton" name="pbRemote">
         <property name="text">
          <string>Remote</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="lCustomBR">
         <property name="text">
          <string>custom BR</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QSpinBox" name="spinBox"/>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="lRxClear">
         <property name="text">
          <string>Rx Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbReceive">
      <property name="title">
       <string>Receive</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pbReceiveClear">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbResetCounter">
           <property name="text">
            <string>Reset Counter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_2"/>
         </item>
         <item>
          <widget class="QLabel" name="lCounter">
           <property name="text">
            <string>Counter = 0</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="1">
            <widget class="QCheckBox" name="cbDec">
             <property name="text">
              <string>Dec</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="cbBin">
             <property name="text">
              <string>Bin</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="cbHex">
             <property name="text">
              <string>Hex</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="rbHex">
             <property name="text">
              <string>HEX</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="rbASCII">
             <property name="text">
              <string>ASCII</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pbStartLog">
           <property name="text">
            <string>StartLog</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbStopLog">
           <property name="text">
            <string>StopLog</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRequestResponse">
           <property name="text">
            <string>Request/Response</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbTransmit">
      <property name="title">
       <string>Transmit</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pbTransmitClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSendFile">
         <property name="text">
          <string>Send File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3"/>
       </item>
       <item>
        <widget class="QCheckBox" name="cbCR">
         <property name="text">
          <string>CR=CR+LF</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbBreak">
         <property name="text">
          <string>Break</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gbMacros">
      <property name="title">
       <string>Macros</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pbSetMacros">
         <property name="text">
          <string>Set Macros</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="1" column="1">
          <widget class="QPushButton" name="pbM14">
           <property name="text">
            <string>M14</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pbM6">
           <property name="text">
            <string>M6</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="pbM3">
           <property name="text">
            <string>M3</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pbM7">
           <property name="text">
            <string>M7</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pbM8">
           <property name="text">
            <string>M8</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QPushButton" name="pbM9">
           <property name="text">
            <string>M9</string>
           </property>
          </widget>
         </item>
         <item row="0" column="9">
          <widget class="QPushButton" name="pbM10">
           <property name="text">
            <string>M10</string>
           </property>
          </widget>
         </item>
         <item row="0" column="10">
          <widget class="QPushButton" name="pbM11">
           <property name="text">
            <string>M11</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pbM2">
           <property name="text">
            <string>M2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pbM5">
           <property name="text">
            <string>M5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pbM1">
           <property name="text">
            <string>M1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="pbM15">
           <property name="text">
            <string>M15</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pbM4">
           <property name="text">
            <string>M4</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pbM13">
           <property name="text">
            <string>M13</string>
           </property>
          </widget>
         </item>
         <item row="0" column="11">
          <widget class="QPushButton" name="pbM12">
           <property name="text">
            <string>M12</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="pbM16">
           <property name="text">
            <string>M16</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QPushButton" name="pbM17">
           <property name="text">
            <string>M17</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QPushButton" name="pbM18">
           <property name="text">
            <string>M18</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QPushButton" name="pbM19">
           <property name="text">
            <string>M19</string>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QPushButton" name="pbM20">
           <property name="text">
            <string>M20</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <widget class="QPushButton" name="pbM21">
           <property name="text">
            <string>M21</string>
           </property>
          </widget>
         </item>
         <item row="1" column="9">
          <widget class="QPushButton" name="pbM22">
           <property name="text">
            <string>M22</string>
           </property>
          </widget>
         </item>
         <item row="1" column="10">
          <widget class="QPushButton" name="pbM23">
           <property name="text">
            <string>M23</string>
           </property>
          </widget>
         </item>
         <item row="1" column="11">
          <widget class="QPushButton" name="pbM24">
           <property name="text">
            <string>M24</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item>
       <widget class="QCheckBox" name="cbCR_2">
        <property name="text">
         <string>+CR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
